public/assets/css/app.css

```diff
@@ -1,562 +1 @@
-/* http://meyerweb.com/eric/tools/css/reset/
```

Comment:
This file is generated, you might consider removing it from the git and adding it to gitignore